### ISSUE TITLE
Feature request: Separate position override for empty tab in "Open link in new tab"

### ISSUE THREAD

**Issue by farkasma** [Robbendebiene/Gesturefy#642]
TL;DR: Using "Open links in new tab", I'd like links to open after the sender, but empty tabs to open at the end, which is currently not possible.

I'm migrating from Vivaldi, where this is the default behavior, as in if there is a link to open, it opens after the sender tab, but if not, and a new tab is to be opened, it goes to the end. I wanted to replicate this, but since the empty tabs also have the active tab as the opener, they fall under the same "Related" group in the eyes of Firefox, and thus use the same default. Easiest way to work around this would be a separate override for empty tabs in settings. Alternative option would be to have no opener id for empty tabs, and then they can use "insertAfterCurrent".
I'd be willing to implement this, just checking if it would be a welcome improvement, and not unwanted clutter.

**Comment by Robbendebiene** [comment]
Hi, you should already be able to do this using the "multi purpose command" It is one of the more advanced commands which can be found at the bottom of the command list. This allows you to combine the "open link in new tab" command (without the empty tab option enabled) and the"new tab command". I hope this works!

Am 10. Juli 2022 19:17:22 MESZ schrieb farkasma ***@***.***>:

**Comment by farkasma** [comment]
Hiya, that solution actually works perfect, thanks for the support! Gonna close this one for now.
